src/fipsy.hpp

```diff
@@ -9,11 +9,202 @@
 /** @brief Fipsy FPGA programmer. */
 class Fipsy {
 public:
-  /**  @brief Fuse table of MachXO2-256. */
-  class FuseTable : public std::bitset<73600> {
+  // Base FuseTable class with a fixed size
+  template <size_t N>
+  /**  @brief Fuse table of - get this from the .jed file e.g. QF73600 for MachXO2-256 and QF343936 for MachXO2-1200 */
+  class FuseTable : public std::bitset<N> {
```

Comment:
This type should stay in the .cpp .
To support different lengths, replace `std::bitset<N>` with [`std::vector<bool>`](https://en.cppreference.com/w/cpp/container/vector_bool), which has a space-efficient specialization.
The vector size is determiend by the **QF** command in the JEDEC file.